### ISSUE TITLE
Transition from BLE devices displayed --> selected --> "signal strength"-feature

### ISSUE THREAD

**Issue by codingwithelijah** [amosproj/amos2022ss05-find-my-hearing-aid#95]
## User story
1. As a user
2. I want / need to be able to first see devices in my area, then select one, and then be directed to the "signal strength"-feature
3. So that I am correctly and intuitively guided in the app

## Further description
precondition: 
* #49 

## Acceptance criteria
* Criterion 1
* Criterion 2
* ...

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
